### PR TITLE
SupportCommand

### PR DESCRIPTION
**Pull request by Ikinon**
# Pull request

 - [ ] I have read and agreed to the [CODE OF CONDUCT](https://github.com/CascadeBot/CascadeBot/blob/master/CODE_OF_CONDUCT.md).
 - [x] What I'm implementing was assigned to me or was not being worked on and is a somewhat requested feature. For reference see our [trello](https://trello.com/b/gUQehy4l/bot).
 - [ ] I have tested all of my changes.
 - [ ] I have followed the style of the rest of the project.
 
## Feature
Support Command

### CODE REVIEW

**Comment by binaryoverload** [comment]
jenkins whitelist

**Comment by Ikinon** [comment]
[;support Command](https://trello.com/c/H1Oa5U2a/10-support-command)

**Comment by weeryan17** [comment]
jenkins test

File: src/main/java/com/cascadebot/cascadebot/commands/core/SupportCommand.java [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+import com.cascadebot.cascadebot.Constants;
+import com.cascadebot.cascadebot.commandmeta.CommandContext;
+import com.cascadebot.cascadebot.commandmeta.ICommandCore;
+import net.dv8tion.jda.core.entities.Member;
+
+public class SupportCommand implements ICommandCore {
+
+    @Override
+    public void onCommand(Member sender, CommandContext context) {
+        context.replyInfo("Cascade Support Server: "+Constants.serverInvite);
```
Comment (weeryan17) [inline src/main/java/com/cascadebot/cascadebot/commands/core/SupportCommand.java]: put spaces between the `+` and the strings

**Pull request merged**